doc/python/configuration-options.md

```diff
@@ -20,9 +20,9 @@ jupyter:
     name: python
     nbconvert_exporter: python
     pygments_lexer: ipython3
-    version: 3.7.3
+    version: 3.7.0
   plotly:
-    description: How to set configuration options of plotly graphs in python.
+    description: How to set the configuration options of figures using Ploty's Python graphing library.
```

Comment:
```suggestion
    description: How to set the configuration options of figures using Plotly Python graphing library.
```

Reply:
@nicolaskruchten do we want to settle for "Plotly Python graphing library" or "Plotly's Python graphing library"? Actually both sound a bit too long to me...